biomage/experiment/info.py

```diff
@@ -0,0 +1,247 @@
+
+            _print_tabbed('error\t', error_string)
+
+        _print_tabbed('start_date', run_details['startDate'])
+        _print_tabbed('stop_date', run_details['stopDate'])
+        _print_tabbed('completed_steps', run_details['completedSteps'])
+        print()
+
+
+def _pretty_print(result):
+
+    WIDTH = 30
+
+    print()
+    print("=" * WIDTH , "EXPERIMENT", "=" * WIDTH, "\n")
+    _format_item(result['info'])
+    print()
+
+    print("=" * WIDTH , "USERS", "=" * WIDTH, "\n")
+    _format_table(result['users'])
+    print()
+
+    print("=" * WIDTH , "SAMPLES", "=" * WIDTH, "\n")
+    if len(result['samples']):
+        _format_table(result['samples'])
+    else:
+        print("No samples uploaded")
+    print()
+
+    print("=" * WIDTH , "PIPELINE RUNNER", "=" * WIDTH, "\n")
+    _format_item(result['pipeline_runner'])
+    print()
+
+    print("=" * WIDTH , "RUNS", "=" * WIDTH, "\n")
+    if len(result['runs']):
+        _format_runs(result['runs'])
+    else:
+        print("Experiment has not been processed")
+
+
+@click.command()
+@click.option(
+    "-e",
+    "--experiment_id",
+    required=True,
+    help="Experiment ID to be copied.",
+)
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=STAGING,
+    show_default=True,
+    help="Input environment to pull the data from.",
+)
+@click.option(
+    "-p",
+    "--aws_profile",
+    required=False,
+    default=DEFAULT_AWS_PROFILE,
+    show_default=True,
+    help="The name of the profile stored in ~/.aws/credentials to use.",
+)
+@click.option(
+    "-o",
+    "--output",
+    required=False,
+    default='false',
+    show_default=True,
+    help="Output result to a format. Supported format: json, yaml",
+)
+def info(
+    experiment_id,
+    input_env,
+    aws_profile,
+    output
+):
+    """
+    Shows the required information related to the experiment.
+    It requires an open tunnel to the desired environment to fetch data from SQL:
+    `biomage rds tunnel -i production`
+
+    E.g.:
+    biomage experiment info -e 2093e95fd17372fb558b81b9142f230e -i production
+    """
+
+    db = init_db(SANDBOX_ID, USER, REGION, input_env, aws_profile)
+
+    info = _get_experiment_info(db, experiment_id)
+
+    pipeline_runner = {
+        'pipeline_runner': "Batch" if info['pod_cpus'] else "Fargate",
+        "pod_cpus": f"{info['pod_cpus'] } cores" if info['pod_cpus'] else "16 cores",
+        "pod_memory": f"{info['pod_memory'] / 1024} GB" if info['pod_memory'] else "28 GB"
+    }
+
+    del info['pod_cpus']
+    del info['pod_memory']
+
+    users = _get_experiment_users(db, experiment_id, input_env)
+    samples = _get_experiment_samples(db, experiment_id)
+    runs = _get_experiment_runs(db, experiment_id)
+
+    result = {
+        "info": info,
+        "pipeline_runner": pipeline_runner,
+        "users": users,
+        "runs": runs,
+        "samples": samples
+    }
+
+    if output == 'json':
+        print(json.dumps(result))
```

Comment:
```suggestion
        print(json.dumps(parsed, indent=4))
```
Use this so that the json will be printed in a readable format.